### PR TITLE
Adds release please for pgai package

### PR DESCRIPTION
**Pull request by alejandrodnm**
Release please will only modify, create release PRs, changelog, git tags, and github releases. For the projects/pgai package.

### CODE REVIEW

**Review by thedodd — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Changelog
```
Comment (jgpruitt) [inline CHANGELOG.md:1]: Is this changelog shared between the python package and the extension?

**Review by jgpruitt — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (alejandrodnm) [reply CHANGELOG.md:1]: No, it's just for changes that affect projects/pgai.

Comment (jgpruitt) [reply CHANGELOG.md:1]: should we nest it under the pgai directory then? We also need a changelog for the extension, or some way to merge the two

Comment (alejandrodnm) [reply CHANGELOG.md:1]: The changelog I think will be created in projectts/pgai/CHANGELOG.md

**Pull request merged**